src/cli/commands/db.ts

```diff
@@ -10,7 +10,9 @@ import { Project } from "../../project/mod.ts";
 export const dbCommand = new Command<GlobalOpts>()
 	.description("Database commands");
 
-dbCommand.action(() => dbCommand.showHelp());
+dbCommand
+	.globalOption("--env <environment:string>", "Managed OpenGB environment used by the Rivet CLI")
```

Comment:
Can we only show this if we detect this is running form the Rivet CLI? Should be pretty easy to pass an env var saying this is a Rivet subcommand.